checkov/cdk/checks/python/ECSClusterContainerInsights.yaml

```diff
@@ -9,9 +9,23 @@ scope:
   languages:
     - python
 definition:
-  pattern: aws_cdk.aws_ecs.CfnCluster(<ANY>)
-  conditions:
-    - not_pattern: |
-        aws_cdk.aws_ecs.CfnCluster(<ANY>, cluster_settings=[<ANY>, {'name': 'containerInsights', 'value': 'enabled'} ,<ANY>], <ANY>)
-    - not_pattern: |
-        aws_cdk.aws_ecs.CfnCluster(<ANY>, cluster_settings=[<ANY>, {'value': 'enabled', 'name': 'containerInsights'} ,<ANY>], <ANY>)
\ No newline at end of file
+  patterns:
+    or:
+      - pattern: aws_cdk.aws_ecs.CfnCluster(<ANY>)
+        conditions:
+          - not_pattern: |
+              aws_cdk.aws_ecs.CfnCluster(<ANY>, cluster_settings=[<ANY>, {'name': 'containerInsights', 'value': 'enabled'} ,<ANY>], <ANY>)
```

Comment:
You could use metadata variables for value enabled or enhanced instead of separate patterns